packages/fluentui/react-bindings/src/context-selector/useContextSelectors.ts

```diff
@@ -31,10 +31,10 @@ export const useContextSelectors = <
 
   const [state, dispatch] = React.useReducer(
     (
-      prevState: readonly [Record<Properties, SelectedValue> /* contextValue */, SelectedValue /* selector(value) */],
+      prevState: readonly [Record<Properties, Value> /* contextValue */, SelectedValue /* selector(value) */],
```

Comment:
```suggestion
      prevState: readonly [
        Value /* contextValue */,
        Record<Properties, SelectedValue> /* { [key]: selector(value) } */,
      ],
```

This actually should be an opposite:

![image](https://user-images.githubusercontent.com/14183168/111300792-c1fd2d80-8651-11eb-9e52-8e5aa3c1cd82.png)

As `[0]` is just `Value` from React's Context, while `[1]` is an object of `SelectedValue`s